docs/reference/errors-and-warnings/NU1901-NU1904.md

```diff
@@ -0,0 +1,37 @@
+---
+title: NuGet Warnings NU1901, NU1902, NU1903, NU1904
```

Comment:
This is interesting. And it works correct from VS because of the f1_keywords I'm guessing?

Is there a chance that this messes with the discoverability of the error codes? Is there any prior art to 1 page for multiple error codes?

Reply:
I don't know if we can test VS before this PR is merged, but I assume it will work.

I know that the compiler's warnings for nullable checks are all on one page: https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/compiler-messages/nullable-warnings.md?plain=1#L4

Reply:
That example is helpful. 

Yeah, we can merge it as is and if then there are F1 problems we can reconsider.